app/models/ticket.rb

```diff
@@ -1,6 +1,9 @@
 class Ticket < ApplicationRecord
+  belongs_to :ticket_desk
   belongs_to :cinema_hall
+  belongs_to :client
   validate :id, :if => :exist?
```

Comment:
Consider if this validation has a general sense? ID is an attribute which has to be generated during creation in DB

Reply:
Well, I was thinking about caching id in session. We cant buy if we "log in", then we have id cached in session and if we buy, id of logged in user is passed as param.